AndroidManifest.xml

```diff
@@ -112,17 +112,29 @@
             android:name=".providers.FileContentProvider"
             android:authorities="@string/authority"
             android:enabled="true"
-            android:exported="false"
+            android:exported="true"
```

Comment:
yes, documents provider is queried by external application, and at that point its outside owncloud permission so exported content provider is required.